README.md

```diff
@@ -103,6 +103,14 @@ $ make test
 
 In order to run the full suite of Acceptance tests, run `make testacc`.
 
+The following ENV variables must be set in your shell prior to running acceptance tests:
```

Comment:
ENV VAR is not the only way, but you can suggest user to use environment variables of SP. And refer to:
https://docs.microsoft.com/en-us/azure/virtual-machines/linux/terraform-install-configure

Reply:
striking out the above comment since it's incorrect, Service Principals are the only supported authentication method for the Acceptance Tests